Lib/fontTools/ttLib/tables/ttProgram.py

```diff
@@ -1067,10 +1069,25 @@ class Program(object):
     def __init__(self):
         pass
 
+    @property
+    def assembly(self):
+        if hasattr(self, "_assembly"):
+            return self._assembly
```

Comment:
just `return self._assembly` is sufficient if you want to raise AttributeError

Reply:
Ah, sure.